client/components/mobile-phone-change/mobile-phone-change.test.js

```diff
@@ -305,4 +324,12 @@ describe("Change Phone Number: corner cases", () => {
     wrapper = await mountComponent(props);
     expect(wrapper.find(Redirect)).toHaveLength(1);
   });
+
+  it("shouldn't redirect if user is active and mobile verificaton is true", async () => {
+    shouldRedirectActiveUserHelper(true);
+  });
+
+  it("should redirect if user is active and mobile verification is false", async () => {
+    shouldRedirectActiveUserHelper(false);
+  });
```

Comment:
just assume mobile verification is true, no need to test when it's false because the component will not be rendered.
So please simplify the test code.